CHANGELOG.md

```diff
@@ -5,6 +5,154 @@ All notable changes to this project will be documented in this file.
+
+The following gives an overview of the changes compared to the previous release. This list is not
+complete, many more internal or minor changes were made, but we tried to only list those changes
+which we think might affect some users directly.
+
+### Algebraic Geometry
+
+#### New features or extended functionality
+
+- [#5403](https://github.com/oscar-system/Oscar.jl/pull/5403) Add `factoring_standard_basis` command which uses `facstd` from `Singular.jl`
+
+### Combinatorics
+
+#### New features or extended functionality
+
+- [#5477](https://github.com/oscar-system/Oscar.jl/pull/5477) Add `multicombinations` to Oscar
+
+#### Improvements or additions to documentation
+
+- [#5431](https://github.com/oscar-system/Oscar.jl/pull/5431) Improve Combinatorics documentation
+
+### Groups
+
+#### New features or extended functionality
+
+- [#5250](https://github.com/oscar-system/Oscar.jl/pull/5250) Add `fixed_points` for `PermGroup` and `PermGroupElem`
+- [#5270](https://github.com/oscar-system/Oscar.jl/pull/5270) Add an iterator of subspaces of finite vector spaces
+- [#5453](https://github.com/oscar-system/Oscar.jl/pull/5453) Add `character_degrees` for a given group or character table, or for given invariants of a finite abelian group and finite field
+
+### Lie Theory
+
+#### Only changes experimental parts of OSCAR
+
+- [#4570](https://github.com/oscar-system/Oscar.jl/pull/4570) Extend functionality to Demazure modules
+
+#### Other fixed bugs
+
+- [#5304](https://github.com/oscar-system/Oscar.jl/pull/5304) Fix `basis_lie_demazure` which wouldn't finish for some inputs
+
+### Number Theory
+
+#### New features or extended functionality
+
+- [#5339](https://github.com/oscar-system/Oscar.jl/pull/5339) Methods for group actions on `ZZLat`
+
+### Polyhedral Geometry
+
+#### New features or extended functionality
+
+- [#5372](https://github.com/oscar-system/Oscar.jl/pull/5372) Add `castelnuovo_excess` and `is_castelnuovo` for lattice polytopes
+
+### Toric Geometry
+
+#### New features or extended functionality
+
+- [#5032](https://github.com/oscar-system/Oscar.jl/pull/5032) Improve vector of minimal exponents method for cohomology computations
+
+### Tropical Geometry
+
+### Changes related to the package GAP
+
+### Changes related to the package Hecke
+
+### Changes related to the package Singular
+
+### Other changes
+
+#### Changes related to serializing data in the MRDI file format
+
+- [#5137](https://github.com/oscar-system/Oscar.jl/pull/5137) Adds OscarDB to experimental
+
+#### New features or extended functionality
+
+- [#5267](https://github.com/oscar-system/Oscar.jl/pull/5267) Add `order_bound` keyword argument to `subgroup_classes`
+- [#5284](https://github.com/oscar-system/Oscar.jl/pull/5284) Implement chamber counting algorithm for toric line bundles
+- [#5288](https://github.com/oscar-system/Oscar.jl/pull/5288) Add `p_rump` for `GAPGroup` and extend `torsion_subgroup` for `GAPGroup` and `WeylGroup`
```

Comment:
This has a "topic: groups" label, but is not sorted into the appropriate section above